packages/react-image/Spec.md

```diff
@@ -0,0 +1,153 @@
+| Additional CSS class name(s) to apply                                   | `className`             | `className`     | matching                      |
+| Specifies the cover style to be used for this image                     | `coverStyle`            | N/A             | v0 missing                    |
+| Replaces the src in case of errors                                      | `errorSrc` [DEPRECATED] | N/A             | _See `onLoadingStateChanged`_ |
+| Determines how the image is scaled and cropped to fit the frame         | `imageFit`              | `fluid`         | partially matching            |
+| Expands image frame to fill its parent container                        | `maximizeFrame`         | `fluid`         | partially matching            |
+| Optional callback for when the image load state changed                 | `onLoadingStateChanged` | N/A             | v0 missing                    |
+| Fades the image in when loaded                                          | `shouldFadeIn`          | N/A             | v0 missing                    |
+| The image starts as visible and is hidden on error                      | `shouldStartVisible`    | N/A             | v0 missing                    |
+| Provides customized styling that will layer on top of the variant rules | `styles`                | `styles`        | matching                      |
+| Theme provided by HOC                                                   | `theme`                 | `variables`     | matching                      |
+| Accessibility behavior overridden by the user                           | N/A                     | `accessibility` | v7/v8 missing                 |
+| An element type to render as (string or component)                      | N/A                     | `as`            | v7/v8 missing                 |
+| An image may be formatted to appear inline with text as an avatar       | N/A                     | `avatar`        | v7/v8 missing                 |
+| An image can appear circular                                            | N/A                     | `circular`      | v7/v8 missing                 |
+| Component design prop                                                   | N/A                     | `design`        | v7/v8 missing                 |
+| Image source URL                                                        | N/A                     | `src`           | v7/v8 missing                 |
+
+### v8 usage
+
+```jsx
+import { Image, ImageFit } from '@fluentui/react/lib/Image';
+
+<Image maximizeFrame imageFit={ImageFit.contain} src="..." alt="Example of the image component in v8" />;
+```
+
+### v0 usage
+
+```jsx
+<Image fluid src="..." alt="Example of the image component in v0" />
+```
+
+## Variants
+
+n/a
+
+## API proposal
+
+See [react-image/Image.types.ts] for the API.
+
+[react-image/image.types.ts]: https://github.com/microsoft/fluentui/blob/master/packages/react-image/src/components/Image/Image.types.ts
```

Comment:
One comment on the types, shouldn't `src` be required?

Reply:
If the component is just rendering an `img` tag underneath is there value in converting the native attr to required props ?

Reply:
also `alt` is optional, I'd say that this is not part of this RFC.  also those typings are redundant as they are covered by "native (react-dom)" `React.ImgHTMLAttributes<HTMLImageElement>` interface

Reply:
addressed in #18972